launcher.py

```diff
@@ -112,6 +114,9 @@ def parse_and_launch():
     viewer = None
     if args.command == 'replay':
         metalog = MetaLog(args.logfile)
+        if conf is None and args.logfile.endswith('.zip'):
+            if DEFAULT_ZIP_CONFIG in zipfile.ZipFile(args.logfile).namelist():
+                conf = Config.loads(zipfile.ZipFile(args.logfile).read(DEFAULT_ZIP_CONFIG))
```

Comment:
No. `if conf is None ...`, i.e. if it was not set originally.

I wanted to have this only in "replay" part while `conf` is also used in "run".

OT I am not sure if "DEFAUTL_ZIP_CONFIG" should not belong to config.py instead??